.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,3 +25,4 @@
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] If I included new strings, I have used `trans.` to make them localizable.
```

Comment:
Do we have docs / a guide for this yet? Even if it is just small, something that appeared in this section of our website https://napari.org/guides/stable/index.html that we could link to here might really help, otherwise I worry people may get to this and not know what to do

Reply:
Where is the source I should update so I can dod it as part of this PR ?

Reply:
those are `.md` files living here https://github.com/napari/napari/tree/master/docs/guides

Reply:
I love the new guide, I think once it is release we'll have a version at `https://napari.org/guides/stable/translations.html`. Do you want to add a link here, so something like

```
- [ ] If I included new strings, I have used `trans.` to make them localizable. For more information
see our [translations guide](https://napari.org/guides/stable/translations.html).
```

Reply:
Will do the fix right away, should be fine I guess :) 

Site build will not take that long